Backpack/TabBarController/Classes/BPKTabBarController.m

```diff
@@ -29,8 +29,29 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+@interface BPKTabBarController ()
+@property (nonatomic, strong) UITabBar *customTabBar;
+@end
+
 @implementation BPKTabBarController
 
+- (instancetype)initWithTabBar:(UITabBar *)tabBar {
+    if (self = [super init]) {
```

Comment:
How does this tab bar actually get used? Don't we need to call something like `[super initWithTabBar: tabBar]`?

Reply:
I'm not sure if I understood the question correctly but basically in order to set a new tabBar the only thing that we need to do is [override the seter](https://stackoverflow.com/a/47560280).
I created a new initialiser just to receive the custom UITabBar instance through there instead of setting a property after init.

Reply:
Ah that makes perfect sense 👍